src/routes/(console)/project-[region]-[project]/sites/+page.svelte

```diff
@@ -53,46 +60,126 @@
             }
         });
     });
+
+    /**
+     * Controls visibility of Sites feature:
+     * - Shown if running on self-hosted
+     * - Shown on cloud only if the organization is Appwrite's.
+     * - Hidden on cloud for any non-Appwrite organization.
+     */
+    $: showSites = !isCloud || $organization.$id === APPWRITE_OFFICIALS_ORG;
+
+    $: isOnWaitlist = isOnWaitlistSites();
+
+    const isDark = $app.themeInUse === 'dark';
+    const isMobile = $isSmallViewport;
+
+    const imgSrc = isDark
```

Comment:
all of those probably should be declarative `$:`, it will not work switch viewport or theme